odex2apk.py

```diff
@@ -38,7 +38,7 @@
 OAT2DEX = os.getenv("OAT2DEX", os.path.join(_dirname, "oat2dex.jar"))
 
 # Supported architectures (first match will be used)
-architectures = ["arm", "arm64", "x86", "x86_64"]
+architectures = ["x86_64", "x86", "arm64", "arm"]
```

Comment:
Ah ok, so it is done to ensure that 64-bit folders are detected before their 32-bit counterparts? So `arm64, arm, x86_64, x86` would also be sufficient?